.github/workflows/ruby.yml

```diff
@@ -74,5 +74,12 @@ jobs:
       - name: Run plugin tests
         if: github.event_name != 'push'
         run: |
-          bundle exec rake test:foreman_leapp
-          bundle exec rake test TEST="test/unit/foreman/access_permissions_test.rb"
+          bundle exec rake test:foreman_leapp -v --trace
+          bundle exec rake test TEST="test/unit/foreman/access_permissions_test.rb" -v --trace
```

Comment:
Why? Doesn't `--trace` just trace what `rake` itself is doing?

Reply:
I admit that I shamelessly copy+pasted from https://github.com/theforeman/foreman_ansible/commit/8631b5aad4cc37ff3b01604729346bd0a6526e2d.

But yes, it turns on invoke/execute tracing, enable full backtrace. I _guess_ it can be useful while debugging what's going on.